bluepysnap/simulation.py

```diff
@@ -85,6 +86,11 @@ def output(self):
         """Access the output section."""
         return self.to_libsonata.output
 
+    @property
+    def input(self):
```

Comment:
Using `inputs` instead of `input` seems more appropriate, since the property returns a dict, and it corresponds to the name used in simulation_config.json.